### DISCUSSION TITLE
Train the model

### DISCUSSION THREAD

**Discussion by tiomaldy** [coqui-ai/TTS#601] [General Q&A]
How i can put my voice for the speaker ? What format need for the dataset,what i need to change in the code ?

**Comment by erogol** [comment]
https://tts.readthedocs.io/en/trainer-api/formatting_your_dataset.html

**Reply by tiomaldy** [reply]
And for train the model ? Step by step guide ?

**Reply by erogol** [reply]
all the up-to-date docs are here for now. https://tts.readthedocs.io/en/dev/ 

I believe you can find it there.
